cm_predict.py

```diff
@@ -200,27 +207,20 @@ def mosaic(self):
         Make a mosaic output from obtained predictions
         Next step: Take into account overlapping argument
         """
-        #self.product_name #tile name
-        # self.big_image_folder#big image (directory)
 
-        self.big_image_product = self.big_image_folder + "/" + self.product_name
-        if not os.path.exists(self.big_image_product):
-            os.mkdir(self.big_image_product)
-
-        # self.predict_folder #working_path
-
-        file_names =['prediction']
+        big_image_product = os.path.join(self.big_image_folder, self.product_name)
+        if not os.path.exists(big_image_product):
+            os.mkdir(big_image_product)
 
         # Create list of prediction images
         image_list = []
-        tile_paths = []
 
         for subfolder in os.listdir(self.prediction_product_path):
             if os.path.isdir(os.path.join(self.prediction_product_path, subfolder)):
                 image_list.append(pathlib.Path(os.path.join(self.prediction_product_path, subfolder, "prediction.png")))
-        #image_list = [y for x in file_names for y in pathlib.Path(self.predict_folder).glob(f'**/{x}*.png')]
 
-        image_list.sort(key=lambda var: get_img_entry_id(var))
+        # Sort images by asc
+        image_list.sort(key = lambda var: get_img_entry_id(var))
```

Comment:
I think usually for keyword arguments, whitespaces are not added around the = symbol.